ios/RNAccengageModule/RNAccengageModule.m

```diff
@@ -8,11 +8,16 @@
 
 #import "RNAccengageModule.h"
 #import <Accengage/Accengage.h>
+#import <React/RCTUtils.h>
+
 
 static NSString *const kRejectCode = @"RNAccengageModule.h";
 static NSString *const kPushRequested = @"pushRequested";
 
 @implementation RNAccengageModule
+    BMA4SInBox      *_inboxMessageList;
```

Comment:
I'd call this _inbox, as it is not a list, it has more functionality.

Reply:
It was renamed to "inbox"